packages/node_modules/@node-red/runtime/lib/module/index.js

```diff
@@ -0,0 +1,163 @@
+/**
```

Comment:
Rename to `subflows/index.js` in line with my other comments.

Reply:
Ranamed to `subflows/index.js`